### PR TITLE
fix: SfCheckbox required text is hidden

### LINKED ISSUE #2380: [BUG] Storybook Molecules Checkbox Required info missing
**Describe the bug**
Required checkbox is not required. Missing information that you need to check this box

**How to reproduce**

1. Open Storybook
1. Go to Molecules -> Checkbox
1. Choose from options or scroll down to "Required"

**Expected behavior**
Next, under or on the top there should be some information that this checkbox is required to click

**Actual behavior**
Checkbox looks like regular checkbox

**Screenshots**
Molecules Checkbox Required info missing

**Devices (please complete the following information):**

* Device: [Desktop]
* OS: [Windows 11]
* Browser: [Chrome]

┆Issue is synchronized with this [Jira Zadanie](https://vsf.atlassian.net/browse/SFUI-129) by [Unito](https://www.unito.io)

### PR DESCRIPTION
**Pull request by RafalDobrowolski**
# Related issue

Closes #2380

# Scope of work
Fix Checkbox Required info missing in the stories.

# Screenshots of visual changes

# Checklist

- [x] No commented blocks of code left
- [x] Run tests and docs
- [x] Self code-reviewed
- [x] Changes documented 

If applicable:

- [ ] I followed [composition rules](https://docs.storefrontui.io/?path=/story/introduction-contributing-guide-code-guidelines--page) for my component
- [ ] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))

### CODE REVIEW

File: packages/vue/src/components/molecules/SfCheckbox/SfCheckbox.stories.js [hunk 1]
```diff
@@ -242,7 +242,6 @@ const Template = (args, { argTypes }) => ({
     style="--checkbox-container-width: 15rem;"
     :name="name"      
     :label="label"
-    :hintMessage="hintMessage"
```
Comment (AdamPawlinski) [inline packages/vue/src/components/molecules/SfCheckbox/SfCheckbox.stories.js:245]: I think we shouldn't remove this prop here. There is a default value of `hintMessage` passed here and it is an empty string. So there is only a need for passing 'hintMessage' value like `Required`.

File: packages/vue/src/components/molecules/SfCheckbox/SfCheckbox.stories.js [hunk 2]
```diff
@@ -309,7 +308,6 @@ export const UseCheckmarkSlot = (args, { argTypes }) => ({
     style="--checkbox-container-width: 15rem;"
     :name="name"
     :label="label"
-    :hintMessage="hintMessage"
```
Comment (AdamPawlinski) [inline packages/vue/src/components/molecules/SfCheckbox/SfCheckbox.stories.js:312]: Like above

**Review by AdamPawlinski — COMMENTED** [review]
You can pass hintMessage value to make required appearing.

File: packages/vue/src/components/molecules/SfCheckbox/SfCheckbox.stories.js (hunk 1 above)
Comment (RafalDobrowolski) [reply packages/vue/src/components/molecules/SfCheckbox/SfCheckbox.stories.js]: done:)

File: packages/vue/src/components/molecules/SfCheckbox/SfCheckbox.stories.js (hunk 2 above)
Comment (RafalDobrowolski) [reply packages/vue/src/components/molecules/SfCheckbox/SfCheckbox.stories.js]: done :)

**Review by AdamPawlinski — APPROVED** [review]
:+1:

**Review by justyna-13 — APPROVED** [review]

**Pull request merged**